gravpm.c

```diff
@@ -276,11 +276,16 @@ void powerspectrum_compute(const int64_t k2, const int kpos[3], pfft_complex * c
     /* Measure power spectrum: we don't want the zero mode.
      * Some modes with k_z = 0 or N/2 have weight 1, the rest have weight 2.
      * This is because of the symmetry of the real fft. */
-    int kint = floor(sqrt(k2));
-    if(kint >= 0 && kint < PowerSpectrum.size) {
+    if(k2 > 0) {
```

Comment:
What is the advantage of using log bins than linear bins? It seems to me going to log removes more information and can always be done as a postprocessing. 
Also, I was thinking of converting the binning to an integer arithmetic to ensure no architecture dependent rounding error changes the binning -- it matters when trying to compare results running from different machines and when there are few modes (very large scale).

This is probably impossible /difficult with log bins.